zipkin docker start error

`docker-compose -f docker-compose.zipkin.yml up` fails both for me and @ZolotarevIgor-mail-ru with
```
zipkin          | java.sql.SQLInvalidAuthorizationSpecException: Could not connect to address=(host=mysql)(port=3306)(type=master) : (conn=662) Access denied for user 'zipkin'@'zipkin.tracing_zipkin' (using password: NO)
zipkin          | 	at org.mariadb.jdbc.internal.util.exceptions.ExceptionFactory.createException(ExceptionFactory.java:66) ~[mariadb-java-client-2.7.1.jar:?]
zipkin          | 	at org.mariadb.jdbc.internal.util.exceptions.ExceptionFactory.create(ExceptionFactory.java:192) ~[mariadb-java-client-2.7.1.jar:?]
zipkin          | 	at org.mariadb.jdbc.internal.protocol.AbstractConnectProtocol.connectWithoutProxy(AbstractConnectProtocol.java:1392) ~[mariadb-java-client-2.7.1.jar:?]
zipkin          | 	at org.mariadb.jdbc.internal.util.Utils.retrieveProxy(Utils.java:635) ~[mariadb-java-client-2.7.1.jar:?]
zipkin          | 	at org.mariadb.jdbc.MariaDbConnection.newConnection(MariaDbConnection.java:150) ~[mariadb-java-client-2.7.1.jar:?]
zipkin          | 	at org.mariadb.jdbc.Driver.connect(Driver.java:89) ~[mariadb-java-client-2.7.1.jar:?]
zipkin          | 	at com.zaxxer.hikari.util.DriverDataSource.getConnection(DriverDataSource.java:121) ~[HikariCP-3.4.5.jar:?]
zipkin          | 	at com.zaxxer.hikari.pool.PoolBase.newConnection(PoolBase.java:358) ~[HikariCP-3.4.5.jar:?]
zipkin          | 	at com.zaxxer.hikari.pool.PoolBase.newPoolEntry(PoolBase.java:206) ~[HikariCP-3.4.5.jar:?]
zipkin          | 	at com.zaxxer.hikari.pool.HikariPool.createPoolEntry(HikariPool.java:477) ~[HikariCP-3.4.5.jar:?]
zipkin          | 	at com.zaxxer.hikari.pool.HikariPool.checkFailFast(HikariPool.java:560) ~[HikariCP-3.4.5.jar:?]
zipkin          | 	at com.zaxxer.hikari.pool.HikariPool.<init>(HikariPool.java:115) ~[HikariCP-3.4.5.jar:?]
zipkin          | 	at com.zaxxer.hikari.HikariDataSource.getConnection(HikariDataSource.java:112) ~[HikariCP-3.4.5.jar:?]
zipkin          | 	at zipkin2.storage.mysql.v1.MySQLStorage.check(MySQLStorage.java:159) ~[zipkin-storage-mysql-v1-2.23.2.jar:?]
zipkin          | 	at zipkin2.server.internal.health.ComponentHealth.ofComponent(ComponentHealth.java:27) ~[classes/:?]
zipkin          | 	at zipkin2.server.internal.health.ZipkinHealthController.lambda$health$1(ZipkinHealthController.java:69) ~[classes/:?]
zipkin          | 	at java.util.concurrent.CompletableFuture$AsyncSupply.run(Unknown Source) ~[?:?]
zipkin          | 	at com.linecorp.armeria.common.RequestContext.lambda$makeContextAware$3(RequestContext.java:502) ~[armeria-1.3.0.jar:?]
zipkin          | 	at io.micrometer.core.instrument.internal.TimedRunnable.run(TimedRunnable.java:44) [micrometer-core-1.6.2.jar:1.6.2]
zipkin          | 	at java.util.concurrent.Executors$RunnableAdapter.call(Unknown Source) [?:?]
zipkin          | 	at java.util.concurrent.FutureTask.run(Unknown Source) [?:?]
zipkin          | 	at java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.run(Unknown Source) [?:?]
zipkin          | 	at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source) [?:?]
zipkin          | 	at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source) [?:?]
zipkin          | 	at io.netty.util.concurrent.FastThreadLocalRunnable.run(FastThreadLocalRunnable.java:30) [netty-common-4.1.54.Final.jar:4.1.54.Final]
zipkin          | 	at java.lang.Thread.run(Unknown Source) [?:?]
zipkin          | Caused by: java.sql.SQLInvalidAuthorizationSpecException: (conn=662) Access denied for user 'zipkin'@'zipkin.tracing_zipkin' (using password: NO)
zipkin          | 	at org.mariadb.jdbc.internal.util.exceptions.ExceptionFactory.createException(ExceptionFactory.java:66) ~[mariadb-java-client-2.7.1.jar:?]
zipkin          | 	at org.mariadb.jdbc.internal.util.exceptions.ExceptionFactory.create(ExceptionFactory.java:187) ~[mariadb-java-client-2.7.1.jar:?]
zipkin          | 	at org.mariadb.jdbc.internal.protocol.AbstractConnectProtocol.authenticationHandler(AbstractConnectProtocol.java:776) ~[mariadb-java-client-2.7.1.jar:?]
zipkin          | 	at org.mariadb.jdbc.internal.protocol.AbstractConnectProtocol.createConnection(AbstractConnectProtocol.java:553) ~[mariadb-java-client-2.7.1.jar:?]
zipkin          | 	at org.mariadb.jdbc.internal.protocol.AbstractConnectProtocol.connectWithoutProxy(AbstractConnectProtocol.java:1387) ~[mariadb-java-client-2.7.1.jar:?]
zipkin          | 	... 23 more
```

--- a/README.md
+++ b/README.md
@@ -181,7 +181,7 @@ version: '3.5'
 
 services:
   storage:
-    image: openzipkin/zipkin-mysql
+    image: openzipkin/zipkin-mysql:1
     container_name: mysql
     networks:
       - zipkin
@@ -191,7 +191,7 @@ services:
   # The zipkin process services the UI, and also exposes a POST endpoint that
   # instrumentation can send trace data to. Scribe is disabled by default.
   zipkin:
-    image: openzipkin/zipkin
+    image: openzipkin/zipkin:1
     container_name: zipkin
     networks:
       - zipkin
@@ -213,7 +213,7 @@ services:
   #
   # For more details, see https://github.com/openzipkin/docker-zipkin-dependencies
   dependencies:
-    image: openzipkin/zipkin-dependencies
+    image: openzipkin/zipkin-dependencies:1
     container_name: dependencies
     entrypoint: crond -f
     networks:

--- a/docker-compose.zipkin.yml
+++ b/docker-compose.zipkin.yml
@@ -5,7 +5,7 @@ version: '3.5'
 
 services:
   storage:
-    image: openzipkin/zipkin-mysql
+    image: openzipkin/zipkin-mysql:1
     container_name: mysql
     networks:
       - zipkin
@@ -15,7 +15,7 @@ services:
   # The zipkin process services the UI, and also exposes a POST endpoint that
   # instrumentation can send trace data to. Scribe is disabled by default.
   zipkin:
-    image: openzipkin/zipkin
+    image: openzipkin/zipkin:1
     container_name: zipkin
     networks:
       - zipkin
@@ -37,7 +37,7 @@ services:
   #
   # For more details, see https://github.com/openzipkin/docker-zipkin-dependencies
   dependencies:
-    image: openzipkin/zipkin-dependencies
+    image: openzipkin/zipkin-dependencies:1
     container_name: dependencies
     entrypoint: crond -f
     networks:

